src/rpc/server.cpp

```diff
@@ -131,6 +131,25 @@ uint256 ParseHashV(const UniValue& v, std::string strName)
     result.SetHex(strHex);
     return result;
 }
+
+int ParseSigHash(std::string strHashType)
+{
+    int nHashType = 0;
+    static std::map<std::string, int> mapSigHashValues = {
```

Comment:
Should `ANYONECANPAY|SINGLE` be possible?

Reply:
why not?

Reply:
Currently it's not.